source/documentation/index.md

```diff
@@ -27,15 +27,13 @@ When you are ready to test your software, first read and understand the [CTC Tra
 
 We strongly advise you to start testing your software for compatibility purposes as soon as possible.
 
-## What is Trader Test?
+## What is the CTC Traders Test Support API?
 
-**Before you can get access to the live production environment, you must check that your software is fully compatible with our CTC Traders API.**
+The CTC Traders Test Support API enables testers to inject transit movement notifications as if they have been sent by the New Computerised Transit System (NCTS) from an office of departure or an office of destination. Until automated responses to trader messages are supported in a trader test environment, the test support API allows for responses to be triggered manually.
 
-Trader Test simulates both the automated responses and also the real life experience where NCTS support staff do the tasks of Border Force personnel.
+Currently, you can inject IE928 messages on valid departure declarations.
 
```

Comment:
Two issues:
- I am not sure about the wording `inject IE928 messages on valid departure declarations`. Can we reword?
- Should `IE928 messages` be `'Positive Acknowledgement' (IE928) messages` or `IE928 ('Positive Acknowledgement') messages`?

Reply:
Any update about these questions?

Reply:
- I've changed "inject on" to "inject for"
- I've switched to using the extended message name format that's used in the DDNTA: ‘Positive Acknowledgement’ E_POS_ACK (IE928) message, ‘Declaration Data’ E_DEC_DAT (IE015) message
- My understanding is that currently only IE928 messages can be injected for valid IE015 messages. Perhaps we shouldn't mention specific support details in the testing guide but point to the Test Support API spec for details.

Reply:
Agreed.